db/migrate/20220325083548_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] OPTIONAL
It's recommended to set bio to text as its purpose is to take a large number of strings or characters